sound/soc/sof/ipc4-topology.c

```diff
@@ -20,6 +20,7 @@
 #define SOF_IPC4_TPLG_ABI_SIZE 6
 
 static DEFINE_IDA(alh_group_ida);
+static DEFINE_IDA(pipeline_ida);
```

Comment:
As it was pointed out by @plbossart on the library loading PR, the IDA is deprecated and XArray should be used, do we want to do that or do a conversion later?

Reply:
Never mind!